api/src/test/java/org/apache/iceberg/transforms/TestTimestampsProjection.java

```diff
@@ -74,22 +74,22 @@ public void assertProjectionStrictValue(PartitionSpec spec, UnboundPredicate<?>
                                           Expression.Operation expectedOp) {
 
     Expression projection = Projections.strict(spec).project(filter);
-    Assert.assertEquals(projection.op(), expectedOp);
+    Assert.assertEquals(expectedOp, projection.op());
   }
 
   public void assertProjectionInclusiveValue(PartitionSpec spec, UnboundPredicate<?> filter,
                                              Expression.Operation expectedOp) {
 
     Expression projection = Projections.inclusive(spec).project(filter);
-    Assert.assertEquals(projection.op(), expectedOp);
+    Assert.assertEquals(expectedOp, projection.op());
   }
 
   public void assertProjectionInclusive(PartitionSpec spec, UnboundPredicate<?> filter,
                                         Expression.Operation expectedOp, String expectedLiteral) {
     Expression projection = Projections.inclusive(spec).project(filter);
     UnboundPredicate<?> predicate = assertAndUnwrapUnbound(projection);
 
-    Assert.assertEquals(predicate.op(), expectedOp);
```

Comment:
More tests need to be added here. I just wanted to get this up for review since I'm going to be out for a couple of days.